subsys/net/conn_mgr/Kconfig

```diff
@@ -58,4 +59,121 @@ config CONNECTIVITY_WIFI_MGMT_APPLICATION
 
 endchoice
 
+config NET_CONNECTION_MANAGER_ONLINE_CONNECTIVITY_CHECK
```

Comment:
Shall we hide all options below under `if NET_CONNECTION_MANAGER_ONLINE_CONNECTIVITY_CHECK / endif` ?

Reply:
There is "depends on" in those options, and they should not be available/seen if checker is not enabled.

Reply:
If you want to stick to `depends on`, then it was missed for `NET_CONNECTION_MANAGER_ONLINE_CHECK_PING` and `NET_CONNECTION_MANAGER_ONLINE_CHECK_HTTP`.